cpp/log/consistent_store.h

```diff
@@ -128,6 +128,8 @@ class ConsistentStore {
 
   void WatchClusterNodeStates(const ClusterNodeStateCallback& cb);
 
+  virtual std::string NodeId() const = 0;
```

Comment:
That's nice and all, but is anyone going to use it soon? If so, I could tolerate it here (but I'd have preferred it to be added with its user), and if not, YAGNI. :wink: